lesson02/elb_asg/main.tf

```diff
@@ -0,0 +1,125 @@
+// Configure AWS Cloud provider
+provider "aws" {
```

Comment:
How are organizing our files, to try to keep consistency? I would leave the `provider` and `terraform` in the main.tf and move other definitions from this file to a different one `asg.tf` or similar

Reply:
we can discuss tomorrow in the standup?